lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -512,7 +512,7 @@ sub sdaf_execute_deployment {
 
     record_info('SDAF exe', "Executing '$args{deployment_type}' deployment: $deploy_command");
     my $rc;
-    my $output_log_file = log_dir() . "/deploy_$args{deployment_type}_attempt.log";
+    my $output_log_file = log_dir() . "/deploy_$args{deployment_type}_attempt.txt";
```

Comment:
I'm fine with it, in other tests we use `.log.txt`. I think it is to be able to directly open them in the editor or even in the browser.

Reply:
Yes. That's where I stole the idea from :D